src/super_gradients/recipes/training_hyperparams/default_train_params.yaml

```diff
@@ -109,15 +109,15 @@ max_train_batches:  # For debug- when not None- will break out of inner train lo
 max_valid_batches:  # For debug- when not None- will break out of inner valid loop
 # (i.e iterating over valid_loader) when reaching this number of batches.
 
-sg_logger: base_sg_logger
-sg_logger_params:
-  tb_files_user_prompt: False # Asks User for Tensorboard Deletion Prompt
-  launch_tensorboard: False
-  tensorboard_port:
-  save_checkpoints_remote: False  # upload checkpoint files to s3
-  save_tensorboard_remote: False  # upload tensorboard files to s3
-  save_logs_remote: False  # upload log files to s3
-  monitor_system: True  # Monitor and write to tensorboard the system statistics, such as CPU usage, GPU, ...
+sg_logger:
+  base_sg_logger:
```

Comment:
How would one "unset" the `base_sg_logger` once it is already in defaults?

I feel this may be the problem if someone would like to use non-default logger:

```yaml
defaults:
  training_hyperparams: default_train_params

training_hyperparams:
  sg_logger:
    wandb_sg_logger:
      ...
```

In this case the resolved config would include:
```
training_hyperparams:
  sg_logger:
    base_sg_logger:
      ...
    wandb_sg_logger:
      ...
```

And that would lead to an exception in factory.

Second use case that would not work is using W&B via Makefile (To keep YAML files intact):

```
WANDB_PARAMS = training_hyperparams.sg_logger=wandb_sg_logger +training_hyperparams.sg_logger_params.api_server=https://wandb.research.deci.ai +training_hyperparams.sg_logger_params.entity=super-gradients training_hyperparams.sg_logger_params.launch_tensorboard=false training_hyperparams.sg_logger_params.monitor_system=true +training_hyperparams.sg_logger_params.project_name=PoseEstimation training_hyperparams.sg_logger_params.save_checkpoints_remote=true training_hyperparams.sg_logger_params.save_logs_remote=true training_hyperparams.sg_logger_params.save_tensorboard_remote=false training_hyperparams.sg_logger_params.tb_files_user_prompt=false

coco2017_yolo_nas_s:
	python src/super_gradients/train_from_recipe.py --config-name=coco2017_yolo_nas_s $(WANDB_PARAMS)

```

Reply:
Strong point right there...
I will think of something.